ui/src/pages/createAccount.tsx

```
@@ -138,13 +132,23 @@ export default function CreateAccountPage() {
         </div>
       </main>
       {errorMsgs().length !== 0 && (
-        <div class="mt-2 mx-10">
-          <ErrorAlert msgs={errorMsgs()} />
+        <div class="mt-4 mx-auto max-w-sm">
+          <div class="bg-red-50 border border-red-500 text-red-600 p-4 rounded-md">
+            <h3 class="font-semibold">Errors:</h3>
+            <ul class="list-disc list-inside">
+              {errorMsgs().map((msg) => (
+                <li>{msg}</li>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)